saturday-afternoon/components/flights/Airlines.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from "react";
+import { Col, Row, Avatar } from "antd";
+import { createFragmentContainer, graphql } from "react-relay";
+import idx from "idx";
+
+import type { Airlines as AirlinesType } from "./__generated__/Airlines.graphql";
+
+type Props = {
+  data: AirlinesType
+};
+
+const Airlines = ({ data: { legs } }: Props) => (
+  <Row>
+    {legs &&
+      legs.map(leg => {
+        if (leg) {
+          const logoUrl = idx(leg, _ => _.airline.logoUrl);
+          const airlineName = idx(leg, _ => _.airline.name) || "";
+          return (
+            <Col key={leg.id} style={{ marginBottom: 10 }}>
```

Comment:
I would use e.g. general `React.Fragment` instead of `Col`, which expect component is placed on specific place in layout

Reply:
Agree that semantically it is not correct. I changed the output so that we have Fragment on top level and each item is wrapped in Row to make sure each item is on new row